content/test/unit/service/validations/Validations.spec.ts

```diff
@@ -401,7 +571,7 @@ function buildEntity(options?: {
   )
   return {
     ...opts,
-    type: EntityType.SCENE
+    type: options?.type ?? EntityType.SCENE
```

Comment:
why do we need this to be added after? Can't it be on the first object with the assignment?

Reply:
good point 👍